chat.revolt.RevoltDesktop.metainfo.xml

```diff
@@ -23,7 +23,22 @@
     </screenshot>
   </screenshots>
   <releases>
-    <release date="2022-03-04" version="1.0.3">
+    <release date="2022-05-10" version="1.0.4">
+      <description>
+        mall quality of life improvements and minor bug fixes:
+        <ul>
+          <li>prevent launching multiple instances.</li>
+          <li>add a tray icon.</li>
+          <li>allow minimising the app to the tray.</li>
+          <li>allow zooming in with "CTRL + =".</li>
+          <li>artifact like shapes on start.</li>
+          <li>add spell check</li>
+          <li>bundle window icon correctly</li>
+          <li>generate DMG rather than ZIP</li>
+        </ul>
+      </description>
+    </release>
+    <release date="2022-03-05" version="1.0.3">
```

Comment:
From what I can see, version [`1.0.3`](https://github.com/revoltchat/desktop/releases/tag/v1.0.3) was released on the fourth of march.
```suggestion
    <release date="2022-03-04" version="1.0.3">
```

Reply:
ah sorry i was editing stuff and forgot i touched that fixed!